pages/posts/content.json

```diff
@@ -1,13 +1,22 @@
+      "author": "Jeff",
+      "img": "/img/LogoCubes.svg"
+    },
+    {
```

Comment:
Can you put post 2 before post 1 so that the blog posts appear as "latest first"?